Sources/FluentPostgreSQL/PostgreSQLDriver.swift

```diff
@@ -2,7 +2,9 @@ import Fluent
 import PostgreSQL
 
 public class PostgreSQLDriver: Fluent.Driver {
-    public var idKey: String = "id"
+    
+    internal static let idKey: String = "id"
+    public var idKey: String = PostgreSQLDriver.idKey
```

Comment:
There being a var here was just a typo (probably got confused by the var on the next line), will make it let.